isilon_create_users.sh

```diff
@@ -47,6 +47,57 @@ function addError() {
 
 function yesno() {
    [ -n "$1" ] && myPrompt=">>> $1 (y/n)? "
+isilon_create_users.sh: 337 lines, 11866 characters
```

Comment:
Oops, looks like something went wrong with your commit... This chunk definitely looks like it will cause a syntax error when running any path of the script.